Automattic/wp-calypso
# Theme showcase: Fix install theme button glitch

When the user refreshes the page and immediately clicks on the install theme button in the Theme Showcase, on an Atomic site, the user is redirected to the Calypso version of the page. This behavior changes after the page loads and the user is then redirected to the WP-Admin Core version (as expected).

Instead of doing that, in this PR, we also check if the features are loaded.

Related to https://github.com/Automattic/wp-calypso/issues/94435

## Proposed Changes

* Link to the WP-Admin theme install page while the page loads

## Why are these changes being made?


* Bug Fix

## Testing Instructions



* Use the Calypso live link
* Go to Theme Showcase on an Atomic site
* Do a hard refresh and immediately click on the install button as quickly as possible (you can reproduce it on WordPress.com prod)
* Notice that you're redirected to the WP-Admin Core page instead of Calypso

## Pre-merge Checklist
